wurst/systems/crafting/CraftingTroll.wurst

```diff
@@ -0,0 +1,156 @@
+    ..addStick()
+    ..addHide()
+    ..addStick()
+    ..generateRecipeString(ABILITY_QM_TENT_KIT)
+
+    let trollHutKit = new ItemRecipeExt(ITEM_TROLL_HUT_KIT, 0, true, true, true, ABILITY_QM_TROLL_HUT_KIT)
+    ..addTent()
+    ..addHide()
+    ..addClayBall()
+    ..generateRecipeString(ABILITY_QM_TROLL_HUT_KIT)
+
+    let mudHutKit = new ItemRecipeExt(ITEM_MUD_HUT_KIT, 0, true, true, true, ABILITY_QM_MUD_HUT_KIT)
+    ..addClayBall()
+    ..addClayBall()
+    ..addClayBall()
+    ..addClayBall()
+    ..generateRecipeString(ABILITY_QM_MUD_HUT_KIT)
+
+    let forgeKit = new ItemRecipeExt(ITEM_FORGE_KIT, 0, true, true, true, ABILITY_QM_FORGE_KIT)
+    ..addFlint()
+    ..addStone()
+    ..addStone()
+    ..addStone()
+    ..generateRecipeString(ABILITY_QM_FORGE_KIT)
+
+    let tanneryKit = new ItemRecipeExt(ITEM_TANNERY_KIT, 0, true, true, true, ABILITY_QM_TANNERY_KIT)
+    ..addStick()
+    ..addStick()
+    ..addStone()
+    ..addStone()
+    ..addClayBall()
+    ..generateRecipeString(ABILITY_QM_TANNERY_KIT)
+
+    let witchDoctorsHutKit = new ItemRecipeExt(ITEM_WITCH_DOCTORS_HUT_KIT, 0, true, true, true, ABILITY_QM_WITCH_DOCTORS_HUT_KIT)
+    ..addStick()
+    ..addManaCrystal(true)
+    ..addStick()
+    ..addManaCrystal(true)
+    ..addStick()
+    ..generateRecipeString(ABILITY_QM_WITCH_DOCTORS_HUT_KIT)
+
+    let workshopKit = new ItemRecipeExt(ITEM_WORKSHOP_KIT, 0, true, true, true, ABILITY_QM_WORKSHOP_KIT)
+    ..addStick()
+    ..addTinder()
+    ..addClayBall()
+    ..addStick()
+    ..generateRecipeString(ABILITY_QM_WORKSHOP_KIT)
+
+    let potKit = new ItemRecipeExt(ITEM_POT_KIT , 0, true, true, true, ABILITY_QM_MIXING_POT_KIT)
+    ..addStick()
+    ..addClayBall()
+    ..addClayBall()
+    ..addClayBall()
+    ..generateRecipeString(ABILITY_QM_MIXING_POT_KIT)
+
+    let armoryKit = new ItemRecipeExt(ITEM_ARMORY_KIT, 0, true, true, true, ABILITY_QM_ARMORY_KIT)
+    ..addStick()
+    ..addIronIngot()
+    ..addIronIngot()
+    ..generateRecipeString(ABILITY_QM_ARMORY_KIT)
+
+    let hatcheryKit = new ItemRecipeExt(ITEM_HATCHERY_KIT, 0, true, true, true, ABILITY_QM_HATCHERY_KIT)
+    ..addStick()
+    ..addStick()
+    ..addStick()
+    ..addStone()
+    ..addStone()
+    ..addStone()
+    ..generateRecipeString(ABILITY_QM_HATCHERY_KIT)
+
+    let teleportBeaconKit = new ItemRecipeExt(ITEM_TELEPORT_BEACON_KIT, 0, true, true, true, ABILITY_QM_TELEPORT_BEACON_KIT)
+    ..addStone()
+    ..addStone()
+    ..addManaCrystal(true)
+    ..addStick()
+    ..addTinder()
+    ..generateRecipeString(ABILITY_QM_TELEPORT_BEACON_KIT)
+
+    let omnidefenderKit = new ItemRecipeExt(ITEM_OMNITOWER_KIT, 0, true, true, true, ABILITY_QM_OMNI_TOWER_KIT)
+    ..addStone()
+    ..addStick()
+    ..addStick()
+    ..addStick()
+    ..generateRecipeString(ABILITY_QM_OMNI_TOWER_KIT)
+
+    // let salve_armor = new ItemRecipeExt(ITEM_SALVE_ARMOR, 1, true, true, true, )
```

Comment:
I understand why you have commented, but at least add a TODO to note it, as well as that the QM don't exist properly so the arguments aren't complete. The variable names also don't follow convention.